src/TesApi.Web/BatchScheduler.cs

```diff
@@ -977,6 +985,142 @@ private ValueTask<bool> HandleTesTaskTransitionAsync(TesTask tesTask, CombinedBa
                 .FirstOrDefault(m => (m.Condition is null || m.Condition(tesTask)) && (m.CurrentBatchTaskState is null || m.CurrentBatchTaskState == combinedBatchTaskInfo.BatchTaskState))
                 ?.ActionAsync(tesTask, combinedBatchTaskInfo, cancellationToken) ?? ValueTask.FromResult(false));
 
+        private async Task<CloudTask> ConvertTesTaskToBatchTaskUsingRunnerAsync(string taskId, TesTask task,
+            (bool ExecutorImage, bool DockerInDockerImage, bool CromwellDrsImage) isPublic,
+            CancellationToken cancellationToken)
+        {
+            ValidateTesTask(task);
+
+            var additionalInputs = await GetAdditionalInputsAsync(task, cancellationToken);
+
+            // TODO: Confirm if the mapping here is correct. This is a carry over from the previous implementation. 
+            var poolHasContainerConfig = !(isPublic.ExecutorImage && isPublic.DockerInDockerImage && isPublic.CromwellDrsImage);
+            var containerCleanupOptions = new RuntimeContainerCleanupOptions(
+                ExecuteDockerRmi: !poolHasContainerConfig,
+                ExecuteDockerPrune: enableBatchAutopool);
```

Comment:
should be `!enableBatchAutopool`. But as discussed, replace both this and the above line with `true`